ultralytics/utils/torch_utils.py

```diff
@@ -324,7 +324,7 @@ def model_info(model, detailed=False, verbose=True, imgsz=640):
 
     flops = get_flops(model, imgsz)
     fused = " (fused)" if getattr(model, "is_fused", lambda: False)() else ""
-    fs = f", {flops:.1f} GFLOPs" if flops else ""
+    fs = f", {flops:.1f} GFLOPs (imgsz={imgsz})" if flops else ""
```

Comment:
```diff
- fs = f", {flops:.1f} GFLOPs (imgsz={imgsz})" if flops else ""
+ fs = f", {flops:.1f} GFLOPs (at imgsz = {imgsz})" if flops else ""
```

Reply:
I thought of adding "at". I guess it makes it clearer.

Reply:
Also, you should be able make it a bit shorter using this:

```py
fs = f", {flops:.1f} GFLOPs (at {imgsz = })" if flops else ""
```

it's [supported in Python 3.8](https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging)

Reply:
Interesting.